EKrut-Server/src/ekrut/server/db/TicketDAO.java

```diff
@@ -0,0 +1,143 @@
+package ekrut.server.db;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import com.mysql.cj.MysqlType;
+
+import ekrut.entity.Ticket;
+
+
+
+//Noy 
+//need to finish - createTicket 
+
+public class TicketDAO {
+	
+	private DBController con;
+	
+	
+	//Constructor - provides connection con to a new TicketDAO so we can use DBcontroller
+	public TicketDAO(DBController con) {
+		this.con = con;
+	}
+	
+	
+	/**  
+	 * Create new ticket line in DB and set its details
+	 * @param Ticket - the ticket that we want to add to DB
+	 * @return true if the addition succeeded , false if failed
+	 */
+	public boolean createTicket(Ticket ticket) {
+		con.beginTransaction();
+		PreparedStatement ps = con.getPreparedStatement("INSERT INTO tickets " +
+                "(ticketID,status,location) " + "VALUES(?,?,?)");
+		try {
+			ps.setInt(1, ticket.getTicketId());
+			ps.setString(2, ticket.getStatus());
+			ps.setString(3, ticket.getEkrutLocation());
+			

```

Comment:
You're not doing anything with the `PreparedStatement`. You probably want to call `ps.executeUpdate()` here.